packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/java/magiclinkcallback.md

```diff
@@ -0,0 +1,8 @@
+```velocity
+#if(${oneTimePassword})
+    <a id="reset-password-link" href="http://localhost:44314/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}">
```

Comment:
Probably doesn't matter, but the Java sample uses 8080 rather than 44314 as ASPNET does

```suggestion
    <a id="reset-password-link" href="http://localhost:8080/magiclink/callback?otp=${oneTimePassword}&state=${request.relayState}">
```